scripts/README.md

```diff
@@ -22,7 +22,7 @@ OPENDAL_VERSION=0.30.2 OPENDAL_VERSION_RC=rc1 ./scripts/release.sh
 ## Check
 
 ```shell
-OPENDAL_VERSION=0.30.2 OPENDAL_VERSION_RC=rc1 ./scripts/check.sh
```

Comment:
It is not quite good that this place is a fixed value? 
It is better to specify that this is an example, and we can add guidance on where to download this file.